.github/workflows/e2e.yml

```diff
@@ -0,0 +1,61 @@
+name: E2E tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # when one test fails, DO NOT cancel the other
+      # containers, because this will kill Cypress processes
+      # leaving the Dashboard hanging ...
+      # https://github.com/cypress-io/github-action/issues/48
+      fail-fast: false
+      matrix:
+        node-version: [14.x]
+        # run 3 copies of the current job in parallel
+        containers: [1, 2, 3]
```

Comment:
What is the benefit of this? Can you split up tests between multiple containers, so they each do a smaller part than 1 big container? Or what's the big idea here?

Reply:
This means there will be 3 instances of Cypress running the tests simultaneously. It's up to Cypress how to split them, but if in the future we have 50 tests, then Cypress can run them grouped in 10-20-20 so we get the results faster.

Reply:
That's really cool! And we don't have to wait around so long then!